doc/fundamentals/distributed_computing.md

```diff
@@ -0,0 +1,115 @@
+# Distributed Computing
+
+Distributing computation across multiple workers is a fundamental part of processing any big data workload. There are two main questions to think about when trying to distribute computation:
+
+1) How many workers to spread computation across?
```

Comment:
I think if you just just use `1.` and `2.` here instead of `1)` etc., it will be rendered as an actual numbered list.  Is that the same for `1)` etc.?